source/Cosmos.HAL2/Audio/AudioBuffer.cs

```diff
@@ -97,12 +88,15 @@ public unsafe void ReadSample(int index, byte* dest)
             if (index >= Size)
                 throw new ArgumentOutOfRangeException(nameof(index));
 
+
             int bufferOffset = index * format.size;
 
             fixed(byte* bufferPtr = buffer) {
-                for (int i = 0; i < format.size; i++) {
-                    *(dest + i) = *(bufferPtr + i);
-                }
+                MemoryOperations.Copy(
+                    dest,
```

Comment:
Could you please write the call in a single line? Style guide is usually to have the call in a single line until it gets very long